frontends/web/src/routes/account/send/utxos.tsx

```diff
@@ -127,6 +128,14 @@ export const UTXOs = ({
                       </span>
                       <FiatConversion alwaysShowAmounts amount={utxo.amount} unstyled noAction/>
                     </div>
+                    <div className={style.date}>
+                      <span className={style.label}>
+                        {t('send.coincontrol.receivedDate')}:
+                      </span>
+                      <span className={style.shrink}>
+                        <Date time={utxo.headerTimestamp} />
```

Comment:
I know this date format is used in the tx list of the account, but there the transactions are sorted by time. Here I find it a bit confusing to see dates like `Saturday, March 22`  or `Wednesday, October 30, 2024`. I think it would be easier to have a `dd/mm/YYYY` format or similar.

Reply:
Please dont' hardcode `dd/mm/YYYY` but use `{parseTimeShort(utxo.headerTimestamp, i18n.language)}`, so the date is localized and short.

```js
new Date().toLocaleString('en-US', {
      month: 'short',
      day: 'numeric',
      year: 'numeric'
    })
"Apr 4, 2025" 

new Date().toLocaleString('de-CH', {
      month: 'short',
      day: 'numeric',
      year: 'numeric'
    })
"4. Apr. 2025" 
```